ninja.py

```diff
@@ -0,0 +1,273 @@
+def Dep(inputs, outputs, rule, params=None):
+    return {
+        'inputs': inputs,
+        'outputs': outputs,
+        'rule': rule,
+        'params': params,
+    }
+
+prefix = 'data/'
+version = '5.3'
```

Comment:
I think you should clarify here that 'version' means the version of the raw data to download, which right now is different from the version number that we're building.